frontend/src/metabase/lib/entities.js

```diff
@@ -664,3 +677,13 @@ export const notify = (opts = {}, subject, verb) =>
 
 export const undo = (opts = {}, subject, verb) =>
   merge({ notify: { subject, verb, undo: true } }, opts || {});
+
+export async function entityCompatibleQuery(entityQuery, dispatch, endpoint) {
+  const request = entityQuery === EMPTY_ENTITY_QUERY ? undefined : entityQuery;
+  const action = dispatch(endpoint.initiate(request, { forceRefetch: true }));
```

Comment:
```suggestion
  const action = dispatch(endpoint.initiate(request, { forceRefetch: true, subscribe: false }));
```
For manually dispatching queries (not mutations) with initiate, RTKQuery auto-subscribes you to the store. As the code stands this would cause a memory leak in the application.